source/widgets/draggablegraphicsview.py

```diff
@@ -40,14 +40,15 @@ def __init__(self, parent):
         QGraphicsView.__init__(self, parent)
 
         self.fPanning = False
+        self.m_ctrl_down = False
```

Comment:
Please rename this var to fCtrlDown, to be consistent with the rest of the code on this file.